src/components/ListTweet.jsx

```diff
@@ -12,6 +20,9 @@ var ListTweet = React.createClass({
                     <i className="fa fa-retweet" /> {this.props.tweet.retweet_count}&nbsp;
                     <i className="fa fa-star" /> {this.props.tweet.favorite_count}
                 </div>
+                <div>
```

Comment:
if tweetMedia is `null` you'll have a useless empty div, should be here if tweetMedia is defined